docs/source/index.rst

```diff
@@ -133,15 +134,15 @@ This work is supported by:
 AiiDA is a NumFOCUS Affiliated Project. Visit `numfocus.org`_ for more information.
 
 .. image:: images/MARVEL.png
-   :height: 100px
+   :height: 75px
    :target: `MARVEL National Centre for Competency in Research`_
 
 .. image:: images/MaX.png
-   :height: 80px
+   :height: 65px
    :target: `MaX European Centre of Excellence`_
 
 .. image:: images/swissuniversities.png
-   :height: 35px
+   :height: 30px
    :target: `swissuniversities P-5 project "Materials Cloud"`_
```

Comment:
Do you know of a way to make these vertically centered (preferably equal height) and equally spaced?

Reply:
You can use
```
.. raw:: html

   <img src="...">
```
but I would like to avoid that if possible